module/VuFind/tests/unit-tests/src/VuFindTest/Navigation/AccountMenuTest.php

```diff
@@ -243,4 +185,44 @@ protected function getOldDefaultMenuConfig(): array
             ],
         ];
     }
+
+    /**
+     * Data provider for testRequiredConfiguration
+     *
+     * @return \Iterator<string, array>
+     */
+    public static function requiredConfigurationProvider(): \Iterator
+    {
+        yield 'Missing group settings' => [
+            ['Account' => []],
```

Comment:
Is it worth adding more specific error message expectations here to further differentiate the test cases?